README.md

```diff
@@ -51,19 +51,20 @@ To install `uv`, follow the [UV Installation Guide](https://docs.astral.sh/uv/ge
 
 
 ```shell
-uv venv openr1 --python 3.11 && source openr1/bin/activate && uv pip install --upgrade pip --link-mode=copy
+uv venv openr1 --python 3.11 && source openr1/bin/activate && uv pip install --upgrade pip
 ```
 
-Next, install vLLM:
+Next, install vLLM and FlasAttention:
 
 ```shell
-uv pip install vllm==0.7.2 --link-mode=copy
+uv pip install vllm==0.7.2
+uv pip install setuptools && uv pip install flash-attn --no-build-isolation
```

Comment:
I had to rebuild the env on the cluster and realised we need this `--no-build-isolation` flag that can be accessed with editable installs. Thus we need to pre-install FA2 before running the editable install